tests/test_use_fields_all.py

```diff
@@ -21,6 +21,7 @@ def _find_fields_node(module_node):
 
 _EXPECTED_ERRORS = {
     "filter_set": {
+        "versions": ">1",
```

Comment:
Should these be `>=2` to be consistent with the `serializer` entry?

Reply:
I don't feel strongly either way as long as it works.

Reply:
Hmm the `use_fields_all` code has `version_specifier = ">=1,<4"` though. Is this test mismatched on purpose?

Reply:
I'm going by the version ranges on `use_fields_all._META_CLASSES`, which has `>=2` for the serializer case but `>1` on the other two for some reason.

Reply:
With the change to the `version_specifier` it was failing the test until I added these to the test. I'm not entirely clear why.

Reply:
> I'm going by the version ranges on `use_fields_all._META_CLASSES`, which has `>=2` for the serializer case but `>1` on the other two for some reason.

I see it now, we have some overlap there with the `version_specifier` that I was looking at. We talked in standup and it makes sense to update them all to `>=2` just for consistency.

I can't put in a suggestion for those lines, are you ok to update them please?

Reply:
Thank you!